tests/integration/sync_test.go

```diff
@@ -1925,6 +1925,11 @@ const complexQueryForDegraphqlRoute = `query SearchPosts($filters: PostsFilters)
 }
 `
 
+const errorConsumerGroupPolicies = "a rate-limiting-advanced plugin with config.consumer_groups\n" +
```

Comment:
Should we capitalise the first letter of error message?

Reply:
This error comes from GDR. It is copied as it is. https://github.com/Kong/go-database-reconciler/blob/main/pkg/utils/utils.go#L29
Also, errors in go are not capitalised by convention. You get lint warnings for the same.